src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -94,7 +94,7 @@ public function getDefinition(): FixerDefinitionInterface
                     ]
                 ),
                 new CodeSample(
-                    <<<'SAMPLE'
+                    <<<'EOD_'
```

Comment:
hm, should it not be EOD? I wonder if rule is covering this edge case?

Reply:
Sure, it does fix it as expected, the `heredoc_closing_marker` rule detected a conflicting substring starting with `EOD` inside the heredoc thus added an extra `_`.